sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -5,9 +5,10 @@
 import sys
 
 import requests
-from SDSCode.database.database import engine
 from SDSCode.database.models import Base
 
+from .database import database as db
```

Comment:
Can you make this consistent with the above import?
```suggestion
from SDSCode.database import database as db
```

Reply:
sure thing since we are not testing this file yet.